ui-admin/src/study/kits/RequestKitsModal.tsx

```diff
@@ -37,8 +38,9 @@ export default function RequestKitsModal({
         Store.addNotification(successNotification(
                     `${response.kitRequests.length} kit requests created`
         ))
+        success = true
       }
-      onSubmit()
+      onSubmit(success)
```

Comment:
I don't think you need the intermediate variable
```suggestion
      onSubmit(!!response.kitRequests.length)
```

Reply:
Done